### PR TITLE
Storage listing

### PR DESCRIPTION
**Pull request by farinalvaro**
This PR solves [RMKT-132](https://rsklabs.atlassian.net/browse/RMKT-132) and [RMKT-135](https://rsklabs.atlassian.net/browse/RMKT-135)

### CODE REVIEW

File: src/components/molecules/CombinedPriceCell.tsx [hunk 1]
```diff
@@ -13,11 +14,11 @@ export interface CombinedPriceCellProps {
 const CombinedPriceCell: FC<CombinedPriceCellProps> = ({
   className = '', divider, price, currency, priceFiat, currencyFiat,
 }) => {
-  const cappedDecimalsFiat = parseFloat(priceFiat).toFixed(4).toString()
+  const cappedDecimalsFiat = priceDisplay(parseFloat(priceFiat), 4)
 
   return (
     <div className={(`priceCell ${className}`).trim()}>
-      <PriceItem key="hola" type="crypto" price={price} currency={currency} />
```
Comment (jurajpiar) [inline src/components/molecules/CombinedPriceCell.tsx:20]: lol well spotted

File: src/utils/parsers.ts [hunk 2]
```diff
@@ -1,14 +1,26 @@
...
-  return big.times(10 ** decimalPlaces)
-    .toString()
+  return big.times(10 ** decimalPlaces).toString()
+}
+
+// parses to wei ignoring the decimals
+export const parseToWei = (src: BigSource): string => {
+  const weiString = parseToBigDecimal(src, -18).toString()
+
+  return weiString.split('.')[0]
```
Comment (jurajpiar) [inline src/utils/parsers.ts:25]:
```suggestion
  return parseInt(parseToBigDecimal(src, -18)).toString()
```
or
```suggestion
  return Math.floor(parseToBigDecimal(src, -18)).toString()
```

File: src/utils/utils.ts [hunk 3]
```diff
@@ -1,5 +1,16 @@
 export const mayBePluralize = (count, noun, suffix = 's'): string => `${count} ${noun}${count !== 1 ? suffix : ''}`
 
+export const priceDisplay = (value: number, maxDecimals = 8): string => value
+  .toFixed(maxDecimals)
+  .toString()
+  .replace(/[.,]00000000$/, '')
```
Comment (jurajpiar) [inline src/utils/utils.ts:6]: what is this suppose to do? Why are the zeros hardcoded? Can this be done better?

File: src/utils/parsers.ts (hunk 2 above)
Comment (farinalvaro) [reply src/utils/parsers.ts:25]: Nice suggestion, thanks 🙂

File: src/utils/utils.ts [hunk 4]
```diff
@@ -1,5 +1,16 @@
...
+
+export const convertGbsToBytes = (gbs: number): number => gbs * 1024 * 1024 * 1024
```
Comment (jurajpiar) [inline src/utils/utils.ts:8]: 1024 * 1024 * 1024 === 1024 ** 3

File: src/utils/utils.ts (hunk 3 above)
Comment (farinalvaro) [reply src/utils/utils.ts:6]: Good point, this aims to remove the unnecessary zeroes after the `,|.` that the `toFixed` adds. Will improve it 🙂

File: src/utils/utils.ts (hunk 4 above)
Comment (farinalvaro) [reply src/utils/utils.ts:8]: You are right about the `**`. About making it more general... meh 😅

**Pull request merged**